lib/std/mem.zig

```diff
@@ -17,6 +17,14 @@ pub const page_size = switch (builtin.cpu.arch) {
         else => 4 * 1024,
     },
     .sparc64 => 8 * 1024,
+    .loongarch32, .loongarch64 => switch (builtin.os.tag) {
```

Comment:
Just noting that #20511 will overhaul this, but I think it's ok to merge this change for now.